src/clientdlg.h

```diff
@@ -42,6 +42,12 @@
 #if QT_VERSION >= QT_VERSION_CHECK( 5, 6, 0 )
 #    include <QVersionNumber>
 #endif
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0) 
```

Comment:
To be checked if all of this is really needed. 
I assume that QPalette was already imported.